src/main/java/dan200/computercraft/ComputerCraft.java

```diff
@@ -106,7 +108,8 @@
 
     // Configuration options
     public static boolean http_enable = true;
-    public static String http_whitelist = "*";
+    public static AddressPredicate http_whitelist = new AddressPredicate( "*" );
+    public static AddressPredicate http_blacklist = new AddressPredicate( );
```

Comment:
Wasn't the point of this feature so we could add localhost and it's various aliases by default?